README.md

```diff
@@ -225,10 +225,8 @@ ros2 launch mara_gazebo mara.launch.py --urdf mara_robot_gripper_140
 <br/>
 
 ## RViz
-### Launching MARA's Simulation
-You need first to [launch MARA in Gazebo](#gazebo).
 
-### Visualizing MARA in RViz2
+First, you will need to [launch MARA in Gazebo](#gazebo) in another terminal.
```

Comment:
you need to fix the header here adding ## and the ToC

Reply:
What's the problem with the header? There is just ## RViz now. ToC is good I think

Reply:
I would say it is fine. Check current master.
https://github.com/AcutronicRobotics/MARA/tree/master#launching-maras-simulation
https://github.com/AcutronicRobotics/MARA/tree/master#visualizing-mara-in-rviz2
are not added.

Reply:
its good. Waiting for CI